portal/api-private-authentication.go

```diff
@@ -36,12 +38,12 @@ func (ps *privateServer) Login(ctx context.Context, in *pb.CLILoginRequest) (*pb
 	// Look for the user on the database by email
 	admin, err := cluster.GetAdminByEmail(appCtx, in.Email)
 	if err != nil {
-		return nil, status.New(codes.Unauthenticated, "Wrong email and/or password.").Err()
+		return nil, status.New(codes.InvalidArgument, "Wrong email and/or password.").Err()
```

Comment:
Unauthenticated was ok, since the nature of the error, is there any specific reason to change it?

Reply:
Technically it's not an authentication problem, or the error is not related to that, the error is related to passing incorrect credentials, so it's an argument related problem